app/components/article-content.js

```diff
@@ -388,6 +390,36 @@ export default Component.extend(
 			}
 		},
 
+		/**
+		 * Handles opening sections when click event occurs on reference
+		 *
+		 * @returns {void}
+		 */
+		handleReferences(event) {
```

Comment:
you can use object destructuring here
`{target}`
and then you can access it 'directly'